static/app/views/settings/organizationTeams/teamSettings/index.tsx

```diff
@@ -47,7 +41,7 @@ export default class TeamSettings extends AsyncView<Props, State> {
     updateTeamSuccess(resp.slug, resp);
     if (id === 'slug') {
       addSuccessMessage(t('Team name changed'));
-      this.props.router.replace(
+      browserHistory.replace(
```

Comment:
Please why did you replace `router` with `browserHistory`? you still get router via props

Reply:
It made tests easier to update as I could use the global jest mocks we have on `browserHistory` instead of having to prepare a stub in the test cases.